### DISCUSSION TITLE
Navigationgroups don't work properly

### DISCUSSION THREAD

**Discussion by m-pernkopf** [filamentphp/filament#5457] [General]
Hy,

I recently figured out, that navigation groups don't work properly on my side in most of the browsers. Where it works in Chrome, Firefox and Edge throw warnings and exceptions coming from Alpine, and I don't know how to handle/fix this.
Example
Warning: 
Alpine Expression Error: this.collapsedGroups is null
Expression: "! ($store.sidebar.groupIsCollapsed(label) && $store.sidebar.isOpen)"

Error:
Uncaught TypeError: this.collapsedGroups is null in app.js
Alpine.store('sidebar', {
    isOpen: Alpine.$persist(true).as('isOpen'),

    collapsedGroups: Alpine.$persist(null).as('collapsedGroups'),

    groupIsCollapsed: function (group) {
        return this.collapsedGroups.includes(group)
    },

What could be the problem here?

Thanks, M

**Comment by danharrin** [comment]
Clear your view cache and delete any published views. If that doesnt work, delete /vendor and reinstall

**Reply by m-pernkopf** [reply]
Thanks for pointing out possible ways to fix it; it worked for me to delete the resources-views-vendor-filament folder. 
I also stumbled over your blog post by coincidence where you get into more detail about this (https://filamentphp.com/blog/publishing-views-in-laravel#staying-safe)

**Reply by davidagoua** [reply]
Thanks it worked for me too

**Comment by Nes-cmd** [comment]
Oooh Dear,  Thank you very much. deleting published filament folder solved the issue on which I bothered for two days. Thanks a lot.

**Reply by bc-devle** [reply]
I deleted it and restore it, and it works. wow.

**Reply by danharrin** [reply]
Please don't restore it, we really dont recommend publishing views

**Comment by puttijayson** [comment]
I've just got the same issue on v3.2.118. How can I fix this?
![image](https://github.com/user-attachments/assets/e9f2e0d8-fa11-42eb-85c3-c82c0864b43d)

**Reply by Andrej-Serafimovski** [reply]
Same isue here. I've been looking everywhere but I cant find anything.

**Reply by Mayank-Grorapid** [reply]
Did Anyone found any solution i am also stuck with same issue have deleted vendor cleared view cache still not working

**Reply by Mayank-Grorapid** [reply]
Found the issue its related to [#14545](https://github.com/filamentphp/filament/issues/14545). Composer update should fix it.

**Reply by danharrin** [reply]
Once updated, clear your browsers local storage

**Reply by abrahampo1** [reply]
> Did Anyone found any solution i am also stuck with same issue have deleted vendor cleared view cache still not working

`composer update`

and

`php artisan filament:upgrade`

That fixed my issue :)

**Reply by Andrej-Serafimovski** [reply]
Thank you so much. Ive been stressing on how to fix this.

**Reply by ringform** [reply]
`php artisan filament:upgrade` fixes the missing brand logo when setting `topbar(false)` in Filament 4.1.
